index.rst

```diff
@@ -84,6 +84,7 @@ That's it! You now have the basic knowledge to start an Akeneo project.
     :titlesonly:
     :hidden:
 
+    migrate_pim/migrate_to_saas
```

Comment:
:thinking: I'm not sure about the place in the dropdown list - for me it should be after the migrate part or before. "How to install the PIM" seems really the first link to have in this kind of documentation to follow the "cycle of life" of the product (Install, then upgrade / migrate)

Reply:
@Doodoune @KarenMayo's feedback was that it should be first in the dropdown to reflect the priority of migrating PaaS clients to SaaS. I agree with you about the "cycle of life" order, but I can also understand the reasons for putting it first.